docs: add neon mcp server with github copilot example (#3460)
neondatabase/website: content/guides/neon-mcp-server-github-copilot-vs-code.md

Let's get started with using the Neon MCP server and GitHub Copilot.

## What You’ll Need

- Node.js (>= v18.0.0) and npm: Download from [nodejs.org](https://nodejs.org/).
- An Azure subscription ([create one](https://azure.microsoft.com/free/cognitive-services) for free)
- Install either the stable or Insiders release of VS Code:
  - [Stable release](https://code.visualstudio.com/download)
  - [Insiders release](https://code.visualstudio.com/insiders)
- Install the [GitHub Copilot](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot), [GitHub Copilot for Azure](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azure-github-copilot), and [GitHub Copilot Chat](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-chat) extensions for VS Code
- [Azure Functions Core Tools](https://learn.microsoft.com/en-us/azure/azure-functions/functions-run-local?tabs=macos%2Cisolated-process%2Cnode-v4%2Cpython-v2%2Chttp-trigger%2Ccontainer-apps&pivots=programming-language-csharp)  (for local testing)

## Connect GitHub Copilot to the Neon MCP Server

### **Create Neon Database**

> Visit the [Neon on Azure Marketplace](https://portal.azure.com/#view/Azure_Marketplace_Neon/NeonCreateResource.ReactView) page and follow the [Create a Neon resource](https://neon.tech/docs/azure/azure-deploy#create-a-neon-resource) guide to deploy Neon on Azure for your subscription. Neon has free plan is more than enough to build proof of concept or kick off a new startup project.

Reviewer: ```suggestion
Visit the [Neon on Azure Marketplace](https://portal.azure.com/#view/Azure_Marketplace_Neon/NeonCreateResource.ReactView) page and follow the [Create a Neon resource](https://neon.tech/docs/azure/azure-deploy#create-a-neon-resource) guide to deploy Neon on Azure for your subscription. Neon offers a [Free plan](https://neon.tech/pricing) that provides more than enough resources to build a proof of concept or kick off a new startup project.
```
